gamemode/modules/inventory/cl_inventory.lua

```diff
@@ -295,6 +295,7 @@ function inv.setHotbarSlot( newIndex )
 end
 
 hook.Add( "InitPostEntity", "RVR_Inventory_HotbarSetup", inv.makeHotbar )
+hook.Add( "OnScreenSizeChanged", "RVR_Inventory_HotbarSetup", inv.makeHotbar )
```

Comment:
Can we check how many times this hook is called

Reply:
once my guy